spec/factories/implementing_organisation.rb

```diff
@@ -1,4 +1,6 @@
 FactoryBot.define do
+  sequence(:reference) { |n| "organisation-#{n}" }
```

Comment:
I am assuming these are declared globally inside FactoryBot so we were relying on the one from `transaction` before it got deleted?

Reply:
Yeah we were... I moved it in here because this is where it's used, I was confused at first because I couldn't figure out where it was declared!